src/main/java/core/basesyntax/utils/DataUtils.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.utils;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class DataUtils {
+    public boolean isDateInRange(String fromDate, String toDate, String validatedDate,
+            String format) {
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern(format);
```

Comment:
If you create a [formatter](https://docs.oracle.com/javase/tutorial/datetime/iso/format.html), make it a constant field.

Reply:
done